src/main/java/strategy/StrategyOperation.java

```diff
@@ -0,0 +1,7 @@
+package strategy;
+
+import model.TransactionDto;
+
+public interface StrategyOperation {
```

Comment:
you have an ambiguous name for the strategy class with this one
rename all of the implementations accordingly
```suggestion
public interface OperationHandler {
```